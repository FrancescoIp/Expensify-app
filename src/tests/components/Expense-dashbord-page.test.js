import React from 'react';
import { shallow } from 'enzyme';
import ExpensDashbordPage from '../../components/expense-dashbord-page';

test('should render Header correctly', () => {
  const wrapper = shallow(<ExpensDashbordPage />);
  expect(wrapper).toMatchSnapshot();
})