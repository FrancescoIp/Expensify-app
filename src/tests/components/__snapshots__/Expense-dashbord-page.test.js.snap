// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div>
  <Connect(ExpenseSumary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
